bool isEnabled
int8 index
string type
float32 param1
float32 param2
float32 holdTime
---
bool success


